js/utils/Logger/Logger.ts

```diff
@@ -2,36 +2,28 @@ module hirespace {
     'use strict';
 
     export class Logger {
-        static log(data: any) {
-            if (hirespace.Debug.getEnvironment() !== 'development') {
-                return false;
-            }
+        static execute(type: string, data: any, notify?: boolean) {
```

Comment:
it originally was but I would not be able to test it externally :( also, it does not change the scope in compiled JS anyway..